Formula/h/hackernews-tui.rb

```diff
@@ -0,0 +1,23 @@
+class HackernewsTui < Formula
+  desc "Terminal UI to browse Hacker News"
+  homepage "https://github.com/aome510/hackernews-TUI"
+  url "https://github.com/aome510/hackernews-TUI/archive/refs/tags/v0.13.4.tar.gz"
+  sha256 "77abc579f58ed33905f00eb4d6047b1d7f30a0b4ed98b44b36ff95648c6ebae0"
+  license "MIT"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args(root: ".", path: "hackernews_tui")
+    bin.install "bin/hackernews_tui" => "hackernews-tui"
```

Comment:
```suggestion
    system "cargo", "install", *std_cargo_args(path: "hackernews_tui")
```

Reply:
We should leave the name binary name the same as what upstream ships.

Reply:
The repository is named `hackernews-TUI`. I think we should follow the principle of least astonishment and name the binary similarly. The homepage and other package managers also use `hackernews-tui` and `hackernews_tui` interchangeably.

Reply:
Why does that take precedence over the repository name? The binary file of `onetrueawk` is `a.out`, but is renamed to a more sane `awk`. In either case, I would prefer having the `name` token be the same as the binary name, as that is the more user friendly choice.

Reply:
cc @Homebrew/core for additional thoughts.

Reply:
Upstream, Arch, MacPorts and pkgsrc all call the binary `hackernews_tui` so it would be odd to diverge from that.

The formula name not matching the binary name is common - even upstream are doing that. I would however be OK with a formula alias here.

Reply:
With or without renaming the formula to `hackernews_tui` as well?

Reply:
@manakbisht without renaming formula, thanks.

Reply:
> Don't you think the user expectation is to use the same name for the command as the package?

No, I don't and the other maintainers don't either. Please stop arguing this point, it's not a good use of anyone's time. Thanks ❤️

Reply:
I am sorry, I was just trying to understand your line of thinking. I may not completely agree with your decision, but I fully respect it. At the end of the day it's far more important to have the package in homebrew than to have my choice of naming.